app/views/shuttering/shutterService/Page2.scala.html

```diff
@@ -0,0 +1,95 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import uk.gov.hmrc.cataloguefrontend.shuttering.{Environment, ShutterStatus, ShutterServiceController, ShutterState}
+@import uk.gov.hmrc.cataloguefrontend.connector.SlugInfoFlag
+@import helper._
+
+@this(viewMessages: ViewMessages)
+
+@( shutter: ShutterServiceController.Shutter
+ )(implicit request         : Request[_]
+          , messagesProvider: MessagesProvider
+          )
+
+@standard_layout(s"Shutter Service") {
+
+  <header>
+    <h1>Shutter Service</h1>
+  </header>
+
+  <div id="service-list">
+
+    <div class="board">
+      <div class="board__body">
+        @helper.form(
+            action =  uk.gov.hmrc.cataloguefrontend.shuttering.routes.ShutterServiceController.step2Post
+          , 'id    -> "shutter-service-form"
+          , 'style -> "padding-left: 20px; padding-right: 20px"
```

Comment:
any value in moving the style tags into css classes? that said, the main .css file is pretty huge so perhaps not